metrics/prometheus.go

```diff
@@ -537,6 +543,26 @@ const (
 	containerEnvPrefix   = "container_env_"
 )
 
+// DefaultContainerLabels implements ContainerLabelsFunc. It exports the
+// container name, first alias, image name as well as all its env and label
+// values.
+func DefaultContainerLabels(container *info.ContainerInfo) map[string]string {
+	set := map[string]string{"id": container.Name}
```

Comment:
So this now handles all labels, including id name etc. I personally like that control for custom impls but it needs to be documented somewhere as well as release noted.